indico/modules/designer/templates/backside_list.html

```diff
@@ -0,0 +1,64 @@
+{% extends 'layout/dialog_base.html' %}
+
+{% macro _render_template_list(templates) %}
+    {% if templates %}
+        <div class="i-box just-group-list">
+            <div class="i-box-content">
+                <ul class="group-list no-content-before badge-backside-list">
+                    {% for tpl in templates | sort(attribute='title') %}
+                        {% set different_size = tpl.data.width|int != width or tpl.data.height|int != height %}
+                        <li {% if different_size %}
+                                class="not-selectable-backside"
+                                title="{% trans %}This template has different dimention than your template{% endtrans %}"
+                            {% endif %}>
+                            <a href="#" class="backside-template"
+                               data-href="{{ url_for('.get_template_data', tpl) }}">
+                                <i class="designer-template-type-icon template-icon-{{ tpl.type.name }}"
+                                   title="{{ tpl.type.title }}"></i>
+                                <span>{{ tpl.title }}</span>
+                            </a>
+                       </li>
+                   {% endfor %}
+                </ul>
+            </div>
+        </div>
+    {% else %}
+        <div class="italic text-not-important">
+            {% trans %}No templates{% endtrans %}
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% block description %}
+    {%- trans -%}
+        Listed below you will find the badge templates that can be used as the back side for your template. Note that
```

Comment:
"Listed below you fill find" sounds a bit strange to me. How about something like "The following badge templates can be used ..."?